Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -400,13 +442,12 @@ def main():
     }
     try:
         if demisto.command() == 'fetch-indicators':
-            indicators = fetch_indicators_command(client, client.indicator_type)
-            # we submit the indicators in batches
-            for b in batch(indicators, batch_size=2000):
-                # remove duplicates due to performance issue -
-                # https://github.com/demisto/etc/issues/25033
-                non_duplicates_dict: Dict[str, Dict] = dict()
-                for indicator in b:
+            indicators_batch = fetch_indicators_command(client, client.indicator_type)
+            non_duplicates_dict: Dict[str, Dict] = dict()
```

Comment:
I think that this means that we save all of the indicators in all of the batches in memory, no?

Reply:
If I understand correctly the meaning of generators each time is stored in memory 2000 indicators.
Because `fetch_indicators_command` returns a generator, the loop that follows it actually calls it over and over again.